Makefile

```diff
@@ -69,28 +69,53 @@ go-deps-clean:
 	$(RM) -r vendor
 
 sonic-gnmi: $(GO_DEPS)
+# advancetls 1.0.0 release need following patch to build by go-1.19
+#	patch -d vendor -p0 < patches/0002-Fix-advance-tls-build-with-go-119.patch
```

Comment:
Will enable this line in next PR: https://github.com/sonic-net/sonic-gnmi/pull/269/files